components/ww3/cime_config/buildnml

```diff
@@ -50,8 +51,13 @@ def buildnml(case, caseroot, compname):
     # Verify wav grid is supported
     #--------------------------------------------------------------------
 
-    wav_grid_supported = ("null", "wQU225EC60to30sp32x36", "wQU225EC60to30", "wQU100", "wQU225EC30to60E2r2")
-    expect(wav_grid in wav_grid_supported, "WAV_GRID '{}' is not supported in ww3. Choose from: '{}'".format(wav_grid,' ,'.join(wav_grid_supported)))
+    wav_grid_supported = ("null",
+                          "wQU225EC60to30sp50x36",
+                          "wQU100sp50x36",
```

Comment:
I think we should remove the wQU100 grid here. It's no longer supported in `config_grids.xml`. I must have overlooked this in the original waves PR - #5047

Reply:
Sure, no problem. I wasn't sure about the wQU100 grid.